### ISSUE TITLE
Add support for pnpm

### ISSUE THREAD

**Issue by SteveALee** [cyclejs-community/create-cycle-app#110]
pnpm is looking pretty solid now and is fast as it links to modules in a cach.

At some point it would be good it create-cycle-app supported it. However I found a couple of problems

# Add option for pnpm
Currently the CLI and `/src/installScripts.js` supports npm or optionally yarn

However it appears you can run `pnmp install` later and to adjust the npm node_modules layout

# Script resolution fails

eg

https://github.com/cyclejs-community/create-cycle-app-flavors/blob/master/packages/cycle-scripts-one-fits-all/index.js#L15

Does not find the script in my case the resolution for `start` was
`C:\projects\pnpmtest\my-app\node_modules\.registry.npmjs.org\cycle-scripts-one-fits-all\2.0.0\node_modules\cycle-scripts-one-fits-all\scripts\start.js` 

That might be an problem with pnpm or else a problem with running pnpm install on a npm installation

**Comment by zkochan** [comment]
So you suggest to run yarn first and then pnpm? pnpm will move folders created by yarn into `node_modules/.ignored`. pnpm won't reuse anything that was installed by yarn, so this solution would be not optimal.

Regarding the error, I don't know what can be the reason, I need to have a deeper look on it.

**Comment by SteveALee** [comment]
@zkochan Thanks for taking a look.

By default it runs npm to install. And I found when I then run `pnpm install` it seemed to move things around to be the pnpm config. But I was concerned it was not 'official' supported. However I found a couple of files were then not found when trying to run the webpack dev server so is not a good solution

Re the error. I rummaged around but got lost. However a better solution is to use the `eject` script so it does no such resolving lookups.

I'll leave this open for others to comment on adding pnpm support. I'd like to use it. I know @staltz has been using it for his work and also likes it.

**Comment by SteveALee** [comment]
Closing as the advantage of pnpn is it links to a local cache of packages and latest npm does the same.

**Comment by zkochan** [comment]
No, npm does not link from the cache. And unlikely will. It copies packages from the cache, like Yarn

**Comment by zkochan** [comment]
I had discussions about this with the npm team. They are not planning to support linking from the cache

**Comment by SteveALee** [comment]
@zkochan  Ah, I was told it does! Thanks for correction.

**Comment by zkochan** [comment]
As a temporary fix for this, I suggest adding a `--skip install` option (the angular-cli has it). Users can just skip installation and manually use pnpm
